src/components/Wowerlay.tsx

```diff
@@ -103,69 +123,97 @@ export const Wowerlay = defineComponent({
       window.removeEventListener('click', handleWindowClick);
     });
 
+    /**
+     * This variable is for watching Popover's animation state.
+     * When we unmount popover if user passed a transition we need to wait
+     * for transition to end to hide Popover.
+     *
+     * When popover animation ends this variable will hide background.
+     */
     const backgroundVisible = ref(props.visible);
```

Comment:
Consider renaming the variable to 'backgroundTransitionFinished'. It's quite descriptive and might render the comment redundant

Reply:
Actualyl comment describes the purpose of the variable. It determines if background visible or not.  I'm updating the comment.

<img width="476" alt="CleanShot 2023-06-02 at 16 25 33@2x" src="https://github.com/wopehq/wowerlay/assets/47540799/612ec1eb-7aab-4b22-a7b4-d3621ebe1f50">